examples/node/app.js

```diff
@@ -1,9 +1,15 @@
+var myHomeServer = "http://localhost:8008";
 var myUserId = "@example:localhost";
 var myAccessToken = "QGV4YW1wbGU6bG9jYWxob3N0.qPEvLuYfNBjxikiCjP";
-var sdk = require("matrix-js-sdk");
-var clc = require("cli-color");
+
+import clc from "cli-color";
+import fs from "fs";
+import readline from "readline";
+import sdk, { ClientEvent, EventType, MsgType, RoomEvent } from "matrix-js-sdk";
+import { KnownMembership } from "matrix-js-sdk/lib/@types/membership.js";
```

Comment:
This is odd but the type isn't exported in `matrix.ts` even though [others are](https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/matrix.ts#L62). Not sure if there's a reason for that?

Reply:
Looks like an oversight to me.

Reply:
Ok. I've moved this to https://github.com/matrix-org/matrix-js-sdk/pull/4405 since fixing it in this PR won't be possible if we stick to an explicit version for the JS SDK dependency.